packages/frontend/src/components/CreateSwap/CreateSwap.tsx

```diff
@@ -92,6 +92,12 @@ const CreateSwap = () => {
   }, [fromToken]);
 
   const handleSwap = () => {
+    if (!quote) {
+      console.log('Attempted to execute swap without a quote.');
```

Comment:
Perhaps we should throw a toast here just in case? Otherwise the user would click, and nothing would happen and they would not know why.

Reply:
This will never run because buttons are disabled if there is no quote. Added this for debugging purposes.

Reply:
Ah my bad